website/pages/docs/nomad-vs-kubernetes/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+page_title: Nomad vs. Kubernetes
+sidebar_title: Nomad vs. Kubernetes
+description: Comparison between Nomad and Kubernetes
+---
+
+# Nomad vs. Kubernetes
+
+Kubernetes is an orchestration system for containers originally designed by Google, now governed by the Cloud Native Computing Foundation (CNCF) and developed by Google, Red Hat, and many others. Kubernetes and Nomad support similar core use cases for application deployment and management, but they differ in a few key ways. Kubernetes aims to provide all the features needed to run Docker-based applications including cluster management, scheduling, service discovery, monitoring, secrets management and more. Nomad only aims to focus on cluster management and scheduling and is designed with the Unix philosophy of having a small scope while composing with tools like Consul for service discovery/service mesh and Vault for secret management.
+
+The following characteristics generally differentiate Nomad from Kubernetes:
+
+## Simplicity
+
+Kubernetes is designed as a collection of more than a half-dozen interoperating services which together provide the full functionality. Coordination and storage is provided by etcd at the core. The state is wrapped by API controllers which are consumed by other services that provide higher level APIs for features like scheduling. Kubernetes supports running in a highly available configuration but is operationally complex to setup.
+
+Nomad is architecturally much simpler. Nomad is a single binary, both for clients and servers, and requires no external services for coordination or storage. Nomad combines a lightweight resource manager and a sophisticated scheduler into a single system. By default, Nomad is distributed, highly available, and operationally simple.
+
+## Flexible Workload Support
+
+While Kubernetes is specifically focused on Docker, Nomad is more general purpose. Nomad supports virtualized, containerized and standalone applications, including Docker, Java, IIS on Windows, Qemu, etc. Nomad is designed with extensible drivers and support will be extended to all common drivers.
```

Comment:
Wording here might use a little update given Kubernetes switching off of "Docker proper" to containerd/CRI.